### DISCUSSION TITLE
List of rcognition results

### DISCUSSION THREAD

**Discussion by DirkWuerdemann** [exadel-inc/CompreFace#1015] [Ideas]
I would like to have a list of recognition results and not only the best hit. Is this possible?

**Comment by pospielov** [comment]
Yes, all you need is to set `prediction_count` param:
https://github.com/exadel-inc/CompreFace/blob/master/docs/Rest-API-description.md#recognize-faces-from-a-given-image

**Reply by DirkWuerdemann** [reply]
Thx
